src/bindings/python/src/compatibility/openvino/__init__.py

```diff
@@ -2,4 +2,53 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
```

Comment:
Please try to return this line and add it to other root init files.
Locally it fixes problem with usage MO/OVC directly from a path.